kernel/posix/pthread_sleep.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <pthread.h>
+/*
+ *  ======== sleep ========
```

Comment:
Such format of comments isn't used elsewhere in Zephyr codebase.

Reply:
Sure i will modify it.